### DISCUSSION TITLE
How to use Api for Logs

### DISCUSSION THREAD

**Discussion by mk148a** [TechnitiumSoftware/DnsServer#625] [Q&A]
Hi there,
I just need api feature using for query logs content. I just need ip adresses in log file

**Comment by ShreyasZare** [comment]
Thanks for asking. You can install the Query Logs (sqlite) DNS App and then use the [Query Logs HTTP API](https://github.com/TechnitiumSoftware/DnsServer/blob/master/APIDOCS.md#query-logs) to get a filtered responses as per your needs.
